internal/pkg/term/selector/local_selector.go

```diff
@@ -125,29 +126,93 @@ func (s *staticSelector) askCron(scheduleValidator prompt.ValidatorFunc) (string
 	return customSchedule, nil
 }
 
+// dockerfileSelector selects from a local file system where a workspace does not necessarily exist.
+type dockerfileSelector struct {
+	prompt        Prompter
+	fs            *afero.Afero
+	workingDirAbs string
+}
+
+// NewDockerfileSelector constructs a DockerfileSelector.
+func NewDockerfileSelector(prompt Prompter, fs afero.Fs) (*dockerfileSelector, error) {
+	workingDirAbs, err := os.Getwd()
+	if err != nil {
+		return nil, fmt.Errorf("get working directory: %w", err)
+	}
+	return &dockerfileSelector{
+		prompt:        prompt,
+		fs:            &afero.Afero{Fs: fs},
+		workingDirAbs: workingDirAbs,
+	}, nil
+}
+
```

Comment:
Can we move these definition to just before `func (s *dockerfileSelector) Dockerfile(selPrompt, notFoundPrompt, selHelp, notFoundHelp string, pathValidator prompt.ValidatorFunc) (string, error)` ?